definitions/artifacts/azure-virtual-network.json

```diff
@@ -40,45 +40,20 @@
         "infrastructure": {
           "title": "Infrastructure configuration",
           "required": [
-            "vnet_id",
+            "id",
```

Comment:
switched this to just "id" to be more consistent w/ all our other artifacts

Reply:
Do we reference this in any downstream bundles?

Reply:
Only in postgres and mysql, which I addressed in the other PRs.  The other stuff just uses `default_subnet_id`